bin/doctest

```diff
@@ -11,12 +11,13 @@ set +f ; shopt -s nullglob
 files=($(get source-files))
 set -f ; shopt -u nullglob
 
+module="$(get module-type)"
 prefix="$(get comment-prefix)"
 opening="$(get opening-delimiter)"
 closing="$(get closing-delimiter)"
```

Comment:
Perhaps `get module-type` should come after `get closing-delimiter`. One day we could add linting to ensure that the variables mentioned in a script's documentation correspond to those it actually uses. This might be easier to do if the variables are listed in the same order in both places.

Reply:
Currently I've put them in order of appearance in the script below.

Reply:
:zap: